File: Simple Fun #145: Table Game/Simple Fun #145: Table Game.js
function tableGame(table) {
  const [c1,c2,c3,c4] = [table[0][0],table[0][table.length-1],table[table.length-1][0],table[table.length-1][table.length-1]]
  if(table[0][1]===c1+c2 && table[table.length-1][1]===c3+c4 && table[1][1]===c1+c2+c3+c4){
    return [c1,c2,c3,c4]
  }else{
    return [-1]
  }
}

File: Simple Fun #145: Table Game/Simple Fun #145: Table Game.dart
List<int> tableGame(List<List<int>> table) {
  int a= table[0][0];
  int b= table[0][2];
  int c= table[2][0];
  int d= table[2][2];
  if((a+b+c+d == table[1][1])&&(a+b == table[0][1])&&(a+c == table[1][0])&&(b+d == table[1][2])&&(c+d == table[2][1])){
    List<int>output=[a,b,c,d];
    return output;
  }else{
  return [-1];
  }
}
